repos/corefx.proj

```diff
@@ -2,7 +2,11 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) -$(Configuration) -buildArch=$(Platform) -portable=false -- /p:ILLinkTrimAssembly=false</BuildCommand>
+    <BuildArguments>-$(Configuration) -buildArch=$(Platform)</BuildArguments>
+    <BuildArguments Condition="'$(PortableBuild)' == ''">$(BuildArguments) -portable=false</BuildArguments>
+    <BuildArguments Condition="'$(PortableBuild)' != ''">$(BuildArguments) -portable=$(PortableBuild)</BuildArguments>
+    <BuildArguments>$(BuildArguments) -- /p:ILLinkTrimAssembly=false</BuildArguments>
+    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildArguments)</BuildCommand>
     <BuildCommand Condition="$(Platform.Contains('arm'))">$(ArmEnvironmentVariables) $(BuildCommand)</BuildCommand>
     <FrameworkBuildCommand>$(ProjectDirectory)/build$(ShellExtension) -$(Configuration) -buildArch=$(Platform) -Framework=netfx -OSGroup=Windows_NT -- /p:ILLinkTrimAssembly=false</FrameworkBuildCommand>
```

Comment:
This needs to be built with `-portable=$(PortableBuild)` as well, I think.

Reply:
Yeah, I wasn't sure if it was appropriate since it didn't have a portable flag passed into it before, but I'll go ahead and give it the same treatment.